.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js: lts/*
+
+script: ./apply.sh
+
+env:
+  global:
```

Comment:
Is this token in the credentials store?

Reply:
No, tempted to leave it that way too as I don't think we want to encourage using the same token for multiple things.

The git history shows who the token 'belongs' to, but I could add a comment if we think that's useful.